clippy_lints/src/casts/cast_possible_truncation.rs

```diff
@@ -165,10 +173,14 @@ pub(super) fn check(
 
     let name_of_cast_from = snippet(cx, cast_expr.span, "..");
     let cast_to_snip = snippet(cx, cast_to_span, "..");
-    let suggestion = format!("{cast_to_snip}::try_from({name_of_cast_from})");
+    let suggestion = if cast_to_snip == "_" {
+        format!("<{cast_to_snip}>::try_from({name_of_cast_from})")
```

Comment:
We could do something like

```rust
format!("{}.try_into()", Sugg::hir(cx, cast_expr, "..").maybe_par())
```

Would be a bit nicer than `<_>::try_from(x)` in my opinion

Reply:
🤔 well, you mean use `try_into()` for that specify scenerio where `cast_to_snip` is a wildcard, or replacing all `try_from` with `try_into` in general? because latter will require the type being known...

Reply:
Just the wildcard case

Reply:
Right, that make sense